pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -147,7 +171,15 @@ def stopping_condition(op: Operator) -> bool:
     # These thresholds are adapted from `lightning_base.py`
     # To avoid building matrices beyond the given thresholds.
     # This should reduce runtime overheads for larger systems.
-    return op.has_matrix and len(op.wires) <= 2 and op.name in _operations
+    if isinstance(op, qml.QFT):
+        return len(op.wires) < 10
+    if isinstance(op, qml.GroverOperator):
+        return len(op.wires) < 13
```

Comment:
Should we check the correctness of these magic numbers?

Reply:
Let's keep it as it is aligned with ``lightning.qubit`` for now. We can work on this later as performance improvement in Q4. We can expect way more tests for MPS than ``lightning.qubit`` since MPS has an additional ``max_mps_bond_dim`` variable to explore.